integration-tests/hibernate-orm-tenancy/discriminator/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+# Database configuration
+quarkus.datasource.db-kind=postgresql
+quarkus.flyway.migrate-at-start=true
+#Reset Flyway metadata at boot, as the database might have been tainted by previous integration tests:
+quarkus.flyway.clean-at-start=true
+
+# Default persistence unit
+quarkus.hibernate-orm.database.generation=none
+quarkus.hibernate-orm.multitenant=discriminator
+
```

Comment:
Do you think you could restore the named datasource ("inventory") and corresponding REST resource/tests we have in other multi-tenancy ITs, just to be sure this will work for named datasources as well?